src/pages/docs/src/codegen/gen-bundlers.js

```diff
@@ -0,0 +1,28 @@
+const { generated, getName, getBundlers } = require('./paths');
+const { render } = require('./markdown');
+const { generateStandardPage } = require('./pages');
+
+function getRoute(name) {
+  return (name && `/bundlers/${name}`) || '';
+}
+
+module.exports = function() {
+  const bundlers = getBundlers();
+
+  const imports = bundlers.map(file => {
+
+    const { mdValue, meta = {} } = render(file, generated);
+    const nameToUse = meta.title.split(' ')[0];
```

Comment:
I would not use `meta` here (more below).

Instead, the name could be inferred from the path, i.e., just take `piral-cli-webpack` / `piral-cli-parcel` (essentially just the plugin name).

Reply:
Names are now beeing generated by the file path (e.g. `piral-cli-webpack` / `piral-cli-parcel`).